Тема 5/Лабораторная работа/task3/main.py

```diff
@@ -1,5 +1,18 @@
-def get_unique_list_numbers() -> list[int]:
-    ...  # TODO написать функцию для получения списка уникальных целых чисел
+from random import randint  # я почитал, но так и не понял, насколько правильно поместить импорт внутри функции?
+
+
+def get_unique_list_numbers() -> list[int]:  # TODO написать функцию для получения списка уникальных целых чисел
```

Comment:
Попробуйте в данной функции в качестве хранилища использовать множество set(), которое в себе хранит уникальные данные и у него есть метод по добавлению элементов, тогда будет достаточно просто сравнивать длину множества с заданной длиной.

Reply:
@EgorOb, спасибо.

Я не очень понял, как направить предложенное решение в качестве ответа на Ваш комментарий. Коммит, пуш сделал. Что дальше делать - не понял. 
Но решение вот такое получается у меня (гораздо более емкое чем было). Правильно ли?

```suggestion
    num_set = set()
    while len(num_set) < 15:
        num_set.add(randint(-10, 10))

    return list(num_set)
```